modules/creating-kubefed-CRs.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * kubefed/installing-kubefed.adoc
+
+[id="creating-kubefed-CRs_{context}"]
+= Creating KubeFed custom resources
+
+You must create the `KubeFedWebHook` and `KubeFed` custom resources (CRs) for {KubeFedProductShortName} deployment.
+The {KubeFedProductName} operator will create the required custom resource definitions (CRDs) for these CRs automatically.
+
+.Prerequisites
+* You will need to have the `oc` CLI tool installed.
+
+.Procedure
+
+. Create a `KubeFedWebHook` resource to instantiate an admission webhook controller for KubeFed. It is recommended to create this CR with `scope: Cluster` to allow the webhook controller to serve API requests coming from all KubeFed control planes within the same cluster.
++
+----
+$ cat <<-EOF | kubectl apply -n <namespace> -f -
```

Comment:
Need to change 'kubectl' to `oc` for creating CR commands but will do this once I have feedback from team to avoid changing the preview during review time.